messaging-client-databus/src/test/java/com/inmobi/messaging/consumer/util/ConsumerUtil.java

```diff
@@ -415,6 +415,54 @@ public static void testTimeoutStats(ClientConfig config, String streamName,
         consumer.getMetrics())).getNumOfTiemOutsOnNext(), 10);
   }
 
+  public static void testConsumerBacklog(ClientConfig config, String streamName,
+      String consumerName, boolean hadoop, Path[] rootDirs,Configuration conf,
+      String testStream, String COLLECTOR_PREFIX) throws Exception {
+    //add one collector
+    FileSystem fs = rootDirs[0].getFileSystem(conf);
+    Path collectorDir = new Path(rootDirs[0].toUri().toString(),
+            "data/" + testStream + "/" + COLLECTOR_PREFIX + "8") ;
+    fs.mkdirs(collectorDir);
+    String dataFile = TestUtil.files[2];
+    TestUtil.setUpCollectorDataFiles(fs, collectorDir, dataFile);
+
+    DatabusConsumer consumer = (DatabusConsumer) createConsumer(hadoop);
+    consumer.init(streamName, consumerName, null, config);
+    Assert.assertEquals(consumer.getTopicName(), streamName);
+    Assert.assertEquals(consumer.getConsumerName(), consumerName);
+    Assert.assertEquals(consumer.getPartitionReaders().size(), 3);
+    Assert.assertEquals(consumer.getPendingDataSize().longValue(),9639l);
```

Comment:
this is the size of test files in collector directory and local stream direcotry. I added an explanation comment for the same.